Collection Rate and Score Analysis

carter-engen opened the pull request:
I have added my code and analysis comments for Collection Rate and Score at Individual and Team levels into Explore.R

carter-engen committed:
Team Collection Rate

I added code to analyze the collection rate of teams with added comments about plots.

carter-engen committed:
Team Collection Rate Plots and Analysis

I added plots to analyze team collection rates. I also included the interaction plot of collection rate v. session order by target (line 671).

jamiahuswalton commented on Explore.R:
```diff
@@ -589,4 +589,111 @@ ggplot(data = plot_data_ind, aes(x = Target, y = Average, color = Dominate.Strat
+            n = length(.data[[dependent_response_team]]), 
+            StEr = sd(.data[[dependent_response_team]]) / sqrt(length(.data[[dependent_response_team]])))
+
+ggplot(data = plot_data_ind, aes(x = Target, y = Average, color = SessionOrder, shape=SessionOrder)) +
```
You are using the plot_data_ind in this plot. When I run the code, Rstudio gives me an error. 

Please change to plot_data_team

jamiahuswalton commented on Explore.R:
```diff
@@ -589,4 +589,111 @@ ggplot(data = plot_data_ind, aes(x = Target, y = Average, color = Dominate.Strat
+# 2 and 3 show a similar pattern of collection rate between targets. This may be a fact of the teams still
+# facing a learning curve. 
+
+ggplot(data = plot_data_ind, aes(x = SessionOrder, y = Average, color = Target, shape=Target)) +
```
Same as above, you used plot_data_ind.

jamiahuswalton commented on Explore.R:
```diff
@@ -589,4 +589,111 @@ ggplot(data = plot_data_ind, aes(x = Target, y = Average, color = Dominate.Strat
+  geom_line(aes(group=SessionOrder, color = SessionOrder)) + 
+  geom_errorbar(aes(ymin = Average - StEr, ymax = Average + StEr), width = 0.2) +
+  labs(y = y_label_rate_team, x = x_label_team, title = title_response_team, fill = "Players")
+# Interaction plot showing team collection rate v. target by session order. It is interesting that sessions
```
Make sure the interpretation is correct. The plot you were viewing when you wrote this may not be the correct plot.

jamiahuswalton commented on Explore.R:
```diff
@@ -589,4 +589,111 @@ ggplot(data = plot_data_ind, aes(x = Target, y = Average, color = Dominate.Strat
+  geom_line(aes(group=Target, color = Target)) + 
+  geom_errorbar(aes(ymin = Average - StEr, ymax = Average + StEr), width = 0.2) +
+  labs(y = y_label_rate_team, x = x_label_teamOrder, title = title_response_team_rate_order, fill = "Players")
+# Interaction plot shoing team collection rate v. session order by Target. This shows how each target responded
```
Make sure the interpretation is correct. The plot you were viewing when you wrote this may not be the correct plot.

jamiahuswalton reviewed (changes requested):
Thanks for doing this!

I found two spots where you use the plot data for the Individual level instead of the team level. Please make those changes and let me know when you update the branch and I will take another look at it.

jamiahuswalton committed:
Made changes

I made a couple changes:

- Fix data being used for two plots
- Minor spelling error

jamiahuswalton commented:
Thanks, Carter!
